test/index.spec.js

```diff
@@ -985,6 +985,47 @@ ava.test('axioms: match(a) => match(filter(a))', (test) => {
 	})
 })
 
+ava.test('.filter() should correctly handle "enum" inside "anyOf"', (test) => {
```

Comment:
We should also be testing a filter match here as well

Reply:
Done!